Scripts/script-DBotPreparePhishingData.yml

```diff
@@ -98,16 +100,25 @@ script: |+
               labels_value_counts[original_label] = labels_value_counts.get(original_label, 0) + 1
               data_texts.append(full_txt)
               data_labels.append(original_label)
-      except Exception:
+      except Exception, e:
           skipped_exception += 1
+          demisto.log("Error parsing: " + str(e))
           continue
   def store_data_as_list(data, listname):
-      data = zlib.compress(data)
-      res = demisto.executeCommand("createList", {"listName": listname,"listData": base64.encodestring(data)})
-      if isError(res[0]):
-          demisto.results(res)
-          return
-      demisto.log("Store file successfully in list " + listname)
+      try:
+          data = zlib.compress(data)
+          res = demisto.executeCommand("createList", {"listName": listname,"listData": base64.encodestring(data)})
+          if isError(res[0]):
+              demisto.results(res[0])
```

Comment:
return_error?

Reply:
no, return error do sys.exit, not need here